.travis.yml

```
@@ -2,7 +2,7 @@ sudo: required
 
 env:
   matrix:
-    - DOCKER_IMAGE=ubuntu:20.04 IGN_COLLECTION="citadel" ROS_DISTRO="foxy"
+    - DOCKER_IMAGE=ubuntu:20.04 IGNITION_VERSION="citadel" ROS_DISTRO="foxy"
```

if we are going to support only one version, I suggest to remove it or add a new entry to the matrix without `IGNITION_VERSION`
```suggestion
    - DOCKER_IMAGE=ubuntu:20.04 ROS_DISTRO="foxy"
```

---

Removed: 5bf9b5c8e3350955191380d56c1f30ca4f8d94db